apps/web/package.json

```diff
@@ -62,19 +63,21 @@
     "react-intl": "^8.0.4",
     "react-router": "^7.11.0",
     "react-to-print": "^3.2.0",
-    "tailwindcss": "^4.1.18",
     "tailwind-merge": "^3.4.0",
     "tailwind-variants": "^3.2.2",
-    "urql": "^5.0.1"
+    "tailwindcss": "^4.1.18",
+    "urql": "^5.0.1",
+    "isbot": "^5"
```

Comment:
Where is `isbot` referenced?

Reply:
`@react-router/dev` mentioned to install this during the setup. I just tried removing it and it doesn't seem to change anyting so maybe it was for some other reason thta got resolved 🤷‍♀️

Reply:
🤷